Utilities/CMake/template-libHttpClient.142.Win32.C.vcxproj

```diff
@@ -43,7 +43,8 @@
     <DefaultLanguage>en-US</DefaultLanguage>
     <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
     <PreferredToolArchitecture>x64</PreferredToolArchitecture>
-    <WindowsTargetPlatformVersion>8.1</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
```

Comment:
which one was the janky win10 version? not this one I assume?

Reply:
it was this, but it's only janky with specific components on VS2017. This is what LibHC had previously, just reverting back to it.